Code/GraphMol/FileParsers/FileWriters.h

```diff
@@ -94,6 +91,17 @@ inline std::string MolToV3KMolBlock(const ROMol &mol, bool includeStereo = true,
   return MolToMolBlock(mol, params, confId);
 }
 
+// \brief generates an MDL v2000 mol block for a molecule
```

Comment:
Will this create broken molblocks for mols with too many atoms or do we throw an error?  Perhaps this should be in the docstring as well.

Reply:
> Will this create broken molblocks for mols with too many atoms or do we throw an error? Perhaps this should be in the docstring as well.

yeah, it will create broken molblocks. I'll add a check for that and throw a ValueError if it happens.